dataactbroker/fileRoutes.py

```diff
@@ -136,6 +136,11 @@ def complete_generation(generation_id):
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.complete_generation(generation_id)
 
+    @app.route("/v1/upload_detached_file/", methods=["POST"])
+    def upload_detached_file():
```

Comment:
Should this route also incorporate the argument checks like below, using `@use_kwargs`?